### DISCUSSION TITLE
Does not use passive listeners to improve scrolling performance

### DISCUSSION THREAD

**Discussion by kumard3** [davidjerleke/embla-carousel#350] [Help] [answered]
I am using embla react in nextJs , and it is causing passive listeners and decreasing the performance of my website , any way to fix it .
![Screen Shot 2022-08-14 at 9 01 26 PM](https://user-images.githubusercontent.com/62591080/184544163-ebaf0a7f-7f24-4843-b673-a3ab340b47bc.png)

**Comment by kumard3** [comment, marked as answer]
it is caused by this lines of code
 ```
 node.addEventListener(type, handler, options);
    listeners.push(function () {
      return node.removeEventListener(type, handler, options);
    });
    return self;
  } 
```

**Reply by davidjerleke** [reply]
Hi @kumard3,

**Update:**

This feature has been released with [v7.0.7](https://github.com/davidjerleke/embla-carousel/releases/tag/v7.0.7). Install the latest Embla version to use this feature. Please select this comment as the answer. Thank you!

Best,
David

**Reply by kumard3** [reply]
Will update and test it , thank you for informing this.

**Reply by davidjerleke** [reply]
@kumard3 Please install 7.0.9 (the latest version) which includes an important bug fix.

Cheers!

**Reply by kumard3** [reply]
![Screenshot 2023-02-06 at 11 00 38 PM](https://user-images.githubusercontent.com/62591080/217043199-4d5ff737-c246-4d82-befe-dfdb3b780cd9.png)
hi @davidjerleke  is it possible to change the `transform: translate3d(266.5px, 0px, 0px);` into `transform: translate3d(0px, 0px, 0px);`

**Reply by davidjerleke** [reply]
Hi @kumard3,

Please create separate discussions for different question so we don't have totally unrelated questions here. This discussion is about passive event listeners.

Thank you for understanding.

Best,
David

**Reply by kumard3** [reply]
ya sure thank you for the response
